### PR TITLE
Added scheduled job to test our http status

### LINKED ISSUE #180: prod environment validation - app is healthy
Accessing the following endpoint returns 200
http://beyond-io.me.uk:5000

Two options:

- GitHub Actions - schedule a workflow
- AWS CloudWatch - Free tear

Acceptance Criteria:
Once a day a prod environment test is enabled

### PR DESCRIPTION
**Pull request by ipolonsk**
close #180 

Created schedule workflow in GitHub actions to check the HTTP code of our production environment, the workflow will run twice a day, will pass the test if it accepts a '200' code status that means 'OK' successful.

### CODE REVIEW

File: .github/workflows/web-access.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Web-Access
```
Comment (lioramilbaum) [inline .github/workflows/web-access.yml:1]: I would suggest we provide a more meaningful name. Something like: prod-health. WDYT?

File: .github/workflows/web-access.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+on:
+  #runs twice a day
+  schedule:
+    - cron:  '0 */12 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (lioramilbaum) [inline .github/workflows/web-access.yml:11]: Use a pinned version for stability

**Review by lioramilbaum — CHANGES_REQUESTED** [review]

File: .github/workflows/web-access.yml (hunk 1 above)
Comment (ipolonsk) [reply .github/workflows/web-access.yml:1]: sounds good

**Review by lioramilbaum — APPROVED** [review]

**Review by IlanZuckerman — APPROVED** [review]
LGTM

**Review by IlanZuckerman — APPROVED** [review]
LGTM

**Pull request merged**